eos/effects/adaptivearmorhardener.py

```diff
@@ -13,6 +14,13 @@
 def handler(fit, module, context):
     damagePattern = fit.damagePattern
 
+    sFit = Fit.getInstance()
```

Comment:
eh... problem here. We can't import services from inside eos.

I mean, obviously we can, but we shouldn't (separation of concerns, etc).

To bypass that limitation, we can set the setting (and any other "engine" settings in the future) on `eos.config`, and then import and use that in the effects that need it. It will take a bit more engineering, but that would be the way to do it to keep eos completely separate.

Or you could set the setting on the fit itself (since we have the fit object in the effects). This is how we do the factor reload setting IIRC. Might be easier and quicker that way.